Added google signin(#27)

Manzee1609 opened the pull request.

himanshusharma89 commented:
@Manzee1609 Can you attach a gif/video and also follow the issue template at most if you can.

Manzee1609 commented:
Ok

On Sat, 20 Mar 2021, 6:40 pm HIMANSHU SHARMA, ***@***.***>
wrote:

> @Manzee1609 <https://github.com/Manzee1609> Can you attach a gif/video
> and also follow the issue template at most if you can.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/himanshusharma89/relic_bazaar/pull/146#issuecomment-803316269>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APPSNLMBCKA3MVAL4MN6QF3TESNENANCNFSM4ZQDK4XQ>
> .
>

Manzee1609 commented:
Related Issue
Fixed #146 

Proposed Changes
Added google signin functionality in login page
Added google signout functionality in dashboard drawer

Additional Information
Used firebase authentication

Screenshots of added work
![image](https://user-images.githubusercontent.com/64956077/111875473-fb21fe80-89bf-11eb-88b6-9be559a88239.png)

![image](https://user-images.githubusercontent.com/64956077/111875467-efced300-89bf-11eb-90e0-0b1a93628c9b.png)


https://user-images.githubusercontent.com/64956077/111875495-168d0980-89c0-11eb-91d5-d494de1b8b4e.mp4
